includes/class-wc-stripe-apple-pay-registration.php

```
@@ -195,10 +195,10 @@ public function register_domain( $secret_key ) {
 			return true;
 
 		} catch ( Exception $e ) {
-			$settings                              = WC_Stripe_Helper::get_stripe_settings();
-			$settings['apple_pay_verified_domain'] = $this->domain_name;
-			$settings['apple_pay_domain_set']      = 'no';
-			WC_Stripe_Helper::update_main_stripe_settings( $settings );
+			$settings    = WC_Stripe_Settings::get_instance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Settings' in method 'register_domain'. (undefined)

(StaticAccess)